src/modules/mount/main.py

```diff
@@ -81,29 +81,27 @@ def mount_partition(root_mount_point, partition, partitions):
     # If a swapfile is used, we also create a subvolume "@swap".
     # Finally we remount all of the above on the correct paths.
     if fstype == "btrfs" and partition["mountPoint"] == '/':
-        has_home_mount_point = False
+        # Root has been mounted to btrfs volume -> create subvolumes from configuration
+        btrfs_subvolumes = libcalamares.job.configuration.get("btrfsSubvolumes") or []
```

Comment:
This returns only [] for some reason.

Reply:
There was a typo in mount.conf and keyerror in this file, fixed now and subvolumes are correctly generated. Not mounted though, besides /@.